web-app/ui/src/services/api.js

```diff
@@ -0,0 +1,167 @@
+import axios from 'axios';
+import { redirect } from 'react-router-dom';
+import {toast} from 'react-toastify';
+import {SessionStore} from './store';
+
+const BASE_URL = 'http://10.87.1.25:5000/api';
+
+const Axios = axios.create({
+  baseURL: BASE_URL,
+  withCredentials: true,
+});
+
+Axios.interceptors.request.use(
+  config => {
```

Comment:
auth wrapper me check karle if user is authenticated or not, if not call this method and authenticate the user, ideally api methods should only call apis and return the response, they should not set the state

Reply:
getToken hai -_-